docs/VARIABLES.md

```diff
@@ -4382,6 +4470,54 @@ Default:  100MB
 
 ***
 
+### kafka_broker_logredactor_enabled
+
+Boolean to enable logredactor for kafka_broker. Defaults to logredactor_enabled. Works only when kafka_broker_custom_log4j is set to true
+
+Default:  "{{ logredactor_enabled }}"
+
+***
+
+### kafka_broker_logredactor_rule_path_local
+
+Location of the kafka_broker logredactor rules file on the control node. This'll be copied to kafka_broker_logredactor_rule_path on the kafka_broker node. 
+
+Default:  "{{ logredactor_rule_path_local }}"
+
+***
+
+### kafka_broker_logredactor_rule_path
+
+Full path and name of the rules file on kafka_broker node. This is the location which will be referenced in the log4j property file when defining the redactor appender.
+
+Default:  "{{ logredactor_rule_path }}"
+
+***
+
+### kafka_broker_logredactor_rule_url
+
+URL address that contains or redirects to the redaction rules. If not "", this will be used to fetch rules. kafka_broker_logredactor_rule_path will be ignored.
```

Comment:
```suggestion
URL that contains or redirects to the redaction rules. If not "", this will be used to fetch rules and kafka_broker_logredactor_rule_path will be ignored.
```